infra/gunicorn_conf.py

```diff
@@ -0,0 +1,76 @@
+# From: https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/2daa3e3873c837d5781feb4ff6a40a89f791f81b/docker-images/gunicorn_conf.py
```

Comment:
Would this be a good usecase for a `GunicornSettings(BaseSettings)` class to handle picking up env vars, providing defaults, and coercing some variables to ints?